pkg/sql/colexec/external/types.go

```diff
@@ -37,11 +38,13 @@ type ExternalParam struct {
 	IgnoreLineTag int
 	// tag indicate the fileScan is finished
 	End       bool
+	Prepare   bool
```

Comment:
What is this prepare?   This need a comment at least and I think, it is a really bad name -- but let me see the comment first.